saige_assoc.py

```diff
@@ -419,11 +432,16 @@ def manage_concurrency_for_job(job: hb.batch.job.Job):
         summary_output_path = (
             f'output_files/summary_stats/{celltype}_all_cis_cv_results.tsv'
         )
-
+        summarise_fake_job = get_batch().new_job(
+            f'Fake summarise CV results for {celltype}'
+        )
+        summarise_fake_job.depends_on(*jobs)
+        summarise_fake_job.always_run()
+        summarise_fake_job.command('echo "fake summarise CV results"')
```

Comment:
I don't think there should be an always_run summary job - making sure each independent gene's chain of jobs starts is one thing, but this job would always run, and would be 'summarising' an incomplete batch. I think it's accurate that this should only run if all jobs up to this point have succeeded?